### PR TITLE
Fix typos in docstrings for Keithley instruments

### PR DESCRIPTION
**Pull request by V0XNIHILI**
See title!

### CODE REVIEW

File: pymeasure/instruments/keithley/keithleyDMM6500.py [hunk 1]
```diff
@@ -815,7 +815,7 @@ def measure_resistance(self, max_resistance=10e6, wires=2):
             self.mode = "resistance 4W"
             self.resistance_4W_range = max_resistance
         else:
-            raise ValueError("Keithley DMM6500 only supports 2 or 4 wire" "resistance meaurements.")
+            raise ValueError("Keithley DMM6500 only supports 2 or 4 wire" "resistance measurements.")
```
Comment (BenediktBurger) [inline pymeasure/instruments/keithley/keithleyDMM6500.py:818]:
```suggestion
            raise ValueError("Keithley DMM6500 only supports 2 or 4 wire resistance measurements.")
```

**Comment by BenediktBurger** [comment]
Hey @V0XNIHILI , thanks for your contribution, fixing typos. It looks good.

You can add yourself to the `authors.txt`.

Then I'll merge your code.

**Comment by V0XNIHILI** [comment]
Done!

**Comment by BenediktBurger** [comment]
Thanks for your work.

**Pull request merged**